simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFragmentFeed.kt

```diff
@@ -741,9 +741,12 @@ class CatalogFragmentFeed : Fragment() {
     fun showAccountPickerAction() {
       // Configure the 'Home Action' in the Toolbar to show the account picker when tapped.
       this.supportActionBar?.apply {
-        setHomeAsUpIndicator(R.drawable.accounts)
-        setHomeActionContentDescription(R.string.catalogAccounts)
-        setDisplayHomeAsUpEnabled(true)
+        // Configure whether or not the user should be able to change accounts
+        if (configurationService.showChangeAccountsUi) {
+            setHomeAsUpIndicator(R.drawable.accounts)
+            setHomeActionContentDescription(R.string.catalogAccounts)
+        }
+        setDisplayHomeAsUpEnabled(configurationService.showChangeAccountsUi)
```

Comment:
I think this would read a little better as an if/else, but it's the same logic:

```kotlin
if (configurationService.showChangeAccountsUi) {
  setHomeAsUpIndicator(R.drawable.accounts)
  setHomeActionContentDescription(R.string.catalogAccounts)
  setDisplayHomeAsUpEnabled(true)
} else {
  setDisplayHomeAsUpEnabled(false)
}
```